src/components/Claims/README.md

```diff
@@ -0,0 +1,38 @@
+## Overview
+
+[#TODO]
+
+This folder contains the code for User Claims - a demo leveraging Ably's [user claims](https://ably.com/docs/api/realtime-sdk/authentication#ably-jwt) feature to allow users to delete their own or other's messages depending on privilege level.
+
+Try out the [live demo](https://examples.ably.dev/claims) of the User Claims.
```

Comment:
should we call this component & the links `User Claims` rather than just `claims`? This will make it consistent with the other two examples.

Reply:
Agreed, https://github.com/ably-labs/realtime-examples/pull/37/commits/5b7f7d40c9d685832593110bcc1e0f5456448090